README.md

```diff
@@ -53,31 +55,56 @@ video = pipe("<my-awesome-prompt>").frames[0]
 export_to_video(video, "output.mp4", fps=8)
 ```
 
-**Note:** For Image-to-Video finetuning, you must install diffusers from [this](https://github.com/huggingface/diffusers/pull/9482) branch (which adds lora loading support in CogVideoX image-to-video) until it is merged.
+**Note:** For Image-to-Video finetuning, you must install diffusers
+from [this](https://github.com/huggingface/diffusers/pull/9482) branch (which adds lora loading support in CogVideoX
+image-to-video) until it is merged.
 
-Below we provide additional sections detailing on more options explored in this repository. They all attempt to make fine-tuning for video models as accessible as possible by reducing memory requirements as much as possible.
+Below we provide additional sections detailing on more options explored in this repository. They all attempt to make
+fine-tuning for video models as accessible as possible by reducing memory requirements as much as possible.
 
 ## Dataset Preparation
 
-Create two files where one file contains line-separated prompts and another file contains line-separated paths to video data (the path to video files must be relative to the path you pass when specifying `--data_root`). Let's take a look at an example to understand this better!
+Create two files: one containing line-separated prompts, and the other containing line-separated paths to video files (
+video file paths must be relative to the path you specify with `--data_root`). Let's understand this better with an
+example!
 
-Assume you've specified `--data_root` as `/dataset`, and that this directory contains the files: `prompt.txt` and `videos.txt`.
+Assume your specified `--data_root` is `/dataset`, and this directory contains the files `prompts.txt` and `videos.txt`.
```

Comment:
I think better to keep as `prompt.txt` for the moment since the example dataset we're using (Disney) contains the filename as `prompt.txt`. If we can change the filename on the HF dataset repository, we can update it here as well to `prompts.txt`

Reply:
Yeah `prompt.txt` should be fine for now. This way we don't have to update the remote artifact (dataset in our case).